SoundSwitch/Localization/TrayIconStrings.resx

```diff
@@ -163,33 +163,30 @@
     <value>Error</value>
   </data>
   <data name="help" xml:space="preserve">
-		<value>Help</value>
+    <value>Help</value>
   </data>
   <data name="donate" xml:space="preserve">
-		<value>Donate</value>
+    <value>Donate</value>
   </data>
   <data name="iconChanger.playback" xml:space="preserve">
-		<value>Playback</value>
+    <value>Playback</value>
   </data>
   <data name="iconChanger.recording" xml:space="preserve">
-		<value>Recording</value>
+    <value>Recording</value>
   </data>
   <data name="iconChanger.none" xml:space="preserve">
-		<value>Default</value>
+    <value>Default</value>
   </data>
   <data name="iconChanger.both" xml:space="preserve">
-		<value>Both</value>
+    <value>Both</value>
   </data>
-  <data name="communityMenu" xml:space="preserve">
-		<value>Community</value>
+  <data name="community" xml:space="preserve">
```

Comment:
The resource key can't be renamed unless you do it in every single resx file.

Else the translation will be broken since it uses the `data="name"` to link them together.

Reply:
Same for all the other one too ^^